package-lock.json

```diff
@@ -6,7 +6,7 @@
   "packages": {
     "": {
       "name": "@briza/illogical",
-      "version": "1.5.6",
+      "version": "1.5.9",
```

Comment:
Will cut a new version at the end. This is just the result of npm install and something that was missed previously.